pkg/measure-config/src/measurefs.go

```diff
@@ -0,0 +1,278 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+const (
+	//TpmDevicePath is the TPM device file path
+	TpmDevicePath   = "/dev/tpmrm0"
+	ConfigPCRIndex  = 14
+	ConfigPCRHandle = tpmutil.Handle(tpm2.PCRFirst + ConfigPCRIndex)
+)
+
+type FileInfo struct {
+	exist          bool
+	measureContent bool
+}
+
+type TpmEvent struct {
+	data string
+	pcr  []byte
+}
+
+// we do not measure content of following files
+// because they are unique for each device
+func getExcludeList() []string {
+	return []string{
+		"/config/tpm_credential",
+		"/config/device.cert.pem",
+		"/config/device.key.pem",
+		"/config/onboard.cert.pem",
+		"/config/onboard.key.pem",
+		"/config/soft_serial",
+	}
+}
+
+func isInExcludeList(path string) bool {
+	for _, file := range getExcludeList() {
+		if file == path {
+			return true
+		}
+	}
+	return false
+}
+
+// theses file may appear later on the device and we record the
+// fact that file exists. during attestation process we can detect
+// this fact by comparing saved and current event log
+func getDangerousList() []string {
+	return []string{
+		"/config/bootstrap-config.pb",
+		"/config/GlobalConfig/override.json",
```

Comment:
Instead of `/config/GlobalConfig/override.json` there should be: `/config/DevicePortConfig/override.json`
Actually, the JSON file can have any name, not necessarily "override" (and in theory there can be multiple JSON files - they all will be loaded as long as they have `json` extension).

Reply:
Note that `config/GlobalConfig/global.json` is OK and EVE would not import any other file from that directory.

Reply:
any file will be measure, we are just loosing the ability to report is a certain file will be removed since we do not know the exact name, but I think this is OK. I'll fix the path

Reply:
I've fixed the path